src/App.jsx

```diff
@@ -4,17 +4,30 @@ import Home from "./components/Home";
 import ArticlePage from "./components/ArticlePage";
 import Nav from "./components/Nav";
 import { Box } from "@mui/material";
+import { UserContext } from "./contexts/UserContext";
+import { useState } from "react";
 
 function App() {
+  const [user, setUser] = useState({
+    username: "grumpy19",
+    name: "Paul Grump",
+    avatar_url:
+      "https://vignette.wikia.nocookie.net/mrmen/images/7/78/Mr-Grumpy-3A.PNG/revision/latest?cb=20170707233013",
+  });
   return (
     <>
-      <Nav />
-      <Box mt={"160px"}>
-        <Routes>
-          <Route path="/" element={<Home />}></Route>
-          <Route path="/articles/:article_id" element={<ArticlePage />}></Route>
-        </Routes>
-      </Box>
+      <UserContext.Provider value={{ user, setUser }}>
```

Comment:
Brill, good to see you making use of a hard coded logged in user, and utilising some context to help with this.